rpm/sample-app-c.spec

```diff
@@ -14,13 +14,11 @@ Simple app written in C
 %autosetup -p1 -n %{name}-%{version}
 
 %build
-cd src
-gcc -o sample-app-c sample-app-c.c -lssusysinfo
+make -C src
 
 %install
 rm -rf %{buildroot}
-mkdir -p %{buildroot}%{_bindir}
-install -m755 src/sample-app-c %{buildroot}%{_bindir}
+DESTDIR=%{buildroot} make -C src install
```

Comment:
`%make_install`?

Reply:
Isn't that affected by JB#58377 as well?

Reply:
nope